File: 백준/Silver/15803. PLAYERJINAH’S BOTTLEGROUNDS/PLAYERJINAH’S BOTTLEGROUNDS.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double x1 = sc.nextInt();
        double y1 = sc.nextInt();
        double x2 = sc.nextInt();
        double y2 = sc.nextInt();
        double x3 = sc.nextInt();
        double y3 = sc.nextInt();

        if(x1==x2 || x2==x3 || x3==x1) {
            if(x1==x2 && x2==x3)
                System.out.println("WHERE IS MY CHICKEN?");
            else
                System.out.println("WINNER WINNER CHICKEN DINNER!");
        }

        else if(y1==y2 || y2==y3 || y3==y1) {
            if(y1==y2 && y2==y3)
                System.out.println("WHERE IS MY CHICKEN?");
            else
                System.out.println("WINNER WINNER CHICKEN DINNER!");
        }

        else {
            if((y2-y1)/(x2-x1)==(y3-y1)/(x3-x1))
                System.out.println("WHERE IS MY CHICKEN?");
            else
                System.out.println("WINNER WINNER CHICKEN DINNER!");
        }
    }
}

File: 백준/Silver/15803. PLAYERJINAH’S BOTTLEGROUNDS/PLAYERJINAH’S BOTTLEGROUNDS.py
arr = [list(map(int, input().split())) for _ in range(3)]
D = (arr[1][0] - arr[0][0])*(arr[2][1] - arr[0][1]) - (arr[1][1] - arr[0][1])*(arr[2][0] - arr[0][0])

if D == 0:
    print('WHERE IS MY CHICKEN?')
else:
    print('WINNER WINNER CHICKEN DINNER!')